carbon-design-system/carbon-addons-iot-react
# fix(fileuploader): add styles to show error message

Closes #1728 

**Summary**

Temporary fix for 1728 until we can pull in the latest Carbon version. I have made a note in the upgrade issue #1702 to remove these styles. 

**Change List (commits, features, bugs, etc)**

- added styles to `_file-uploader.scss

**Acceptance Test (how to verify the PR)**

- Go to file uploader [story](https://deploy-preview-1824--carbon-addons-iot-react.netlify.app/?path=/story/fileuploader--fileuploaderitem)
- click on the 'invalid' knob
- see error message show up.
